string region_type # UNDEFINED, RECTANGLE, ELLIPSE, SPIRAL
string status # NO_REGION, SEARCHING, DONE
bool publishing # is the node currently publishing data
int32 current_waypoint # current waypoint from 0 to total waypoints
int32 total_waypoints # total number of generated waypoints